src/DefaultEditor.js

```diff
@@ -49,38 +49,23 @@ class DefaultEditor extends Component {
               show
             />
 
-            <DataSelector
-              label="Labels"
-              attr="labels"
-              clearable={false}
-              hasBlank
-            />
+            <DataSelector label="Labels" attr="labels" clearable hasBlank />
```

Comment:
I'm not sure about hasBlank, but to me, Data Selectors should all be clearable, because the user should be able to completely reset them, or not use some of the available ones.

The one that I think should not be clearable, is the trace selector dropdown. We always want for a trace type to be selected, and we have a default one, which is scatter, so there should be no situation where that dropdown appears, but no trace type is selected

Reply:
OK, so I vote we remove the `clearable` and `hasBlank` props.

Reply:
Yep I agree we should make that a default prop type then we can take out `clearable` all together (except `<TraceSelector clearable={false} />`)

Reply:
Though TraceSelector could itself default to `clearable={false}` so that even that is unnecessary.

Reply:
ok, hasBlank doesn't seem to be doing anything, I'll remove it